From a5d65f2670a664197a26309810d6a2c258158e1f Mon Sep 17 00:00:00 2001
From: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Date: Tue, 20 Nov 2012 18:59:34 +0100
Subject: [PORT FROM MAIN]wifi: check if supplicant is running before starting it

BZ: 66748

Wrong property was used to check if wpa_supplicant was running.
Fix this

Change-Id: I1abb044dfb4fa4e7c739e73544b3b3fe051732a3
Orig-Change-Id: I2aac65171c681a4606cc7bd6c32b093418000b9e
Category: device enablement
Domain: CWS.WIFI-Common
Origin: upstream
Upstream-Candidate: yes
Signed-off-by: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
---
 wifi/wifi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wifi/wifi.c b/wifi/wifi.c
index 4f681e9..11357fe 100644
--- a/wifi/wifi.c
+++ b/wifi/wifi.c
@@ -495,7 +495,7 @@ int wifi_start_supplicant(int p2p_supported)
     }
 
     /* Check whether already running */
-    if (property_get(supplicant_name, supp_status, NULL)
+    if (property_get(supplicant_prop_name, supp_status, NULL)
             && strcmp(supp_status, "running") == 0) {
         return 0;
     }
-- 
1.7.4.1

